volume/volume.go

```diff
@@ -3,6 +3,7 @@ package volume
 import (
 	log "github.com/Sirupsen/logrus"
 
+	"github.com/akutz/golf"
```

Comment:
Hi @clintonskitson,

See this [note](https://github.com/emccode/rexray/pull/67/files#r39933268).